src/reducers/versions.tsx

```diff
@@ -1138,9 +1150,10 @@ export const goToRelativeDiff = ({
   };
 };
 
-const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
-  state = initialState,
-  action,
+const reducer = (
+  state: VersionsState | undefined = initialState,
+  action: ActionType<typeof actions>,
+  { _getPrunedExpandedPaths = getPrunedExpandedPaths } = {},
 ): VersionsState => {
```

Comment:
I did not figure out how to reuse the generic type `Reducer<S,A>` here, hoping it is not a problem...